.travis.yml

```diff
@@ -44,6 +44,7 @@ jobs:
       install: skip
 
       deploy:
+        distributions: "sdist bdist_wheel"
```

Comment:
At the very least, we should definitely add `bdist_wheel` and publish a universal wheel, instead of only the sdist.

Reply:
do we really need to though? we don't have compiled code, so it would pretty much be a sdist.

Reply:
the nice part about a wheel is that it never needs to run setup.py, it just lays down the file, so it is faster to install as well.